google-api-client/src/main/java/com/google/api/client/googleapis/json/GoogleJsonResponseException.java

```diff
@@ -141,6 +141,10 @@ public static GoogleJsonResponseException from(JsonFactory jsonFactory, HttpResp
     }
     // message
     StringBuilder message = HttpResponseException.computeMessageBuffer(response);
+    if (message.length() > 0) {
+      message.append(StringUtils.LINE_SEPARATOR);
```

Comment:
The message should be platform independent. Just use `\n`.

Reply:
Done.